pilot/pkg/bootstrap/server.go

```diff
@@ -257,6 +262,8 @@ func NewServer(args *PilotArgs) (*Server, error) {
 		return nil, err
 	}
 
+	s.initCACertsWatcher()
```

Comment:
can this by done inside createIstioCa(). We should avoid crowding the main bootstrap function as much as possible. Also, this should only be done if CA is enabled

Reply:
Ok, I will change that.

Reply:
Done